bugtool/cmd/configuration.go

```diff
@@ -211,7 +211,7 @@ func copyCiliumInfoCommands(cmdDir string, k8sPods []string) []string {
 	// Most of the output should come via debuginfo but also adding
 	// these ones for skimming purposes
 	ciliumCommands := []string{
-		"cilium debuginfo",
+		"cilium debuginfo -o json",
```

Comment:
Can you keep the old line and add `"cilium debuginfo -o json"`? It's really hard for me reading long json files, I like to start with a more human readable format and then go to json if I need to.

Reply:
@aanm output is now saved in both formats. Had to do some restructuring of the command to achieve this.